File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { CustomTempMailSDK } from '..'


describe('exists', async () => {

  test('test-mode', () => {
    const testsdk = CustomTempMailSDK.test()
    equal(testsdk instanceof CustomTempMailSDK, true,
      'CustomTempMailSDK.test() must return a client synchronously')
  })

})

File: lua/test/exists_test.lua
-- CustomTempMail SDK exists test

local sdk = require("custom-temp-mail_sdk")

describe("CustomTempMailSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
